homeassistant/components/netatmo/services.yaml

```diff
@@ -46,6 +46,29 @@ set_preset_mode_with_end_datetime:
       selector:
         datetime:
 
+set_scheduled_room_temperature:
```

Comment:
this sets the temperature for all rooms in a "zone", or for a single room? if it applies to multiple rooms then maybe we should rename the service to reflect that.

Reply:
it sets the temperature for the targeted thermostat. I call it like this:

```
data:
  zone_name: Comfort
  temperature: "{{ target_comfort_temperature }}"
target:
  entity_id: "{{ thermostat_entity_id }}"
alias: Set temperature for Comfort mode
action: netatmo.set_scheduled_room_temperature
```

Reply:
the method was renamed to `set_scheduled_thermostat_temperature`